Jenkinsfile

```diff
@@ -0,0 +1,203 @@
+            defaultValue: 'scf',
+        )
+        string(
+            name: 'S3_PREFIX',
+            description: 'AWS S3 prefix to publish to',
+            defaultValue: 'scf/config/',
+        )
+        credentials(
+            name: 'DOCKER_CREDENTIALS',
+            description: 'Docker credentials used for publishing',
+            defaultValue: 'cred-docker-scf',
+        )
+        string(
+            name: 'FISSILE_DOCKER_REGISTRY',
+            defaultValue: '',
+            description: 'Docker registry to publish to',
+        )
+        string(
+            name: 'FISSILE_DOCKER_ORGANIZATION',
+            defaultValue: 'splatform',
+            description: 'Docker organization to publish to',
+        )
+    }
+
+    environment {
+        FISSILE_DOCKER_REGISTRY = "${params.FISSILE_DOCKER_REGISTRY}"
+        FISSILE_DOCKER_ORGANIZATION = "${params.FISSILE_DOCKER_ORGANIZATION}"
+    }
+
+    stages {
+        stage('reset') {
+            when {
+                expression { return params.RESET }
+            }
+            steps {
+                checkout(
+                    $class: scm.$class,
+                    userRemoteConfigs: scm.userRemoteConfigs,
+                    branches: scm.branches,
+                    doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+                    submoduleCfg: scm.submoduleCfg,
+                    browser: scm.browser,
+                    gitTool: scm.gitTool,
+                    extensions: [
+                        [$class: 'CheckoutOption', timeout: 30],
+                        [$class: 'CloneOption', timeout: 30],
+                        [$class: 'SubmoduleOption', recursiveSubmodules: true, timeout: 60],
+                        [$class: 'CleanBeforeCheckout'],
+                    ],
+
+                )
+            }
+        }
+        stage('clean') {
+            when {
+                expression { return params.CLEAN }
+            }
+            steps {
+                sh '''
+                    docker images --format="{{.Repository}}:{{.Tag}}" | \
+                        grep -E '/scf-|/uaa-|^uaa-role-packages:|^scf-role-packages:' | \
+                        xargs --no-run-if-empty docker rmi
+                '''
+            }
+        }
+        stage('checkout') {
+            when {
+                expression { return ! params.SKIP_CHECKOUT }
+            }
+            steps {
+                sh '''
+                    git config --global --replace-all submodule.fetchJobs 0
+                '''
+                checkout scm
+            }
+        }
+        stage('tools') {
+            steps {
+                ansiColor('xterm') {
+                    sh '''
+                        set -e +x
+                        source ${PWD}/.envrc
+                        set -x
+                        unset HCF_PACKAGE_COMPILATION_CACHE
+                        make ${FISSILE_BINARY}
```

Comment:
Because that's what is listed in the Makefile as the target.  I don't actually remember what it is, but using the variable works :D  The target just runs `install_tools.sh`.  (The _right_ `install_tools.sh`, after Hart's changes for vagrant box v2.0.6)